mmv1/products/redis/Cluster.yaml

```diff
@@ -123,6 +136,25 @@ properties:
     default_from_api: true
     immutable: true
     required: false
+  - !ruby/object:Api::Type::NestedObject
+    name: zoneDistributionConfig
```

Comment:
nit: You could set `immutable: true` on the top-level field and it would [mark all subfields as immutable too](https://googlecloudplatform.github.io/magic-modules/develop/field-reference/#immutable). It depends on whether this block is likely to contain a mixture of immutable and non-immutable fields in future or not.

Reply:
ACK